prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -219,7 +219,7 @@ def prompt_processor() -> Any:
             structured_output, variable_names, output, promptx
         )
 
-        doc_id = ToolIndex.generate_file_id(
+        doc_id = tool_index.generate_file_id(
```

Comment:
@chandrasekharan-zipstack did we change the `generate_file_id` from static method to instance method

Reply:
Saw the change in other PR